### PR TITLE
Database Summary - Update Autoload Table

### PR DESCRIPTION
**Pull request by timotei-litespeed**
PR created after suggestions: https://github.com/litespeedtech/lscache_wp/issues/717

### CODE REVIEW

File: src/db-optm.cls.php [hunk 1]
```diff
@@ -309,10 +309,15 @@ public function autoload_summary()
 	{
 		global $wpdb;
 
-		$summary = $wpdb->get_row("SELECT SUM(LENGTH(option_value)) AS autoload_size,COUNT(*) AS autload_entries FROM `$wpdb->options` WHERE autoload='yes'");
+		$autoloads = version_compare( get_bloginfo( 'version' ), '6.6.0', '>=' ) ?
```
Comment (mukeshpanchal27) [inline src/db-optm.cls.php:312]: Instead of version compare can we use `function_exists`?

Comment (szepeviktor) [inline src/db-optm.cls.php:312]:
```suggestion
		$autoloads = function_exists( 'wp_autoload_values_to_autoload' ) ? 
```

Comment (timotei-litespeed) [reply src/db-optm.cls.php]: Done

Comment (timotei-litespeed) [reply src/db-optm.cls.php]: Thank you, changed

**Pull request merged**